### ISSUE TITLE
FIM System tests. 0210: Scheduled scan

### ISSUE THREAD

**Issue by DFolchA** [wazuh/wazuh-qa#553]
Hello team,

This issue is related to #493 . It will contain the advances related to the creation of scenario 210.

### Objectives

Implement scenario 210.

### Tasks

- [x] Create ossec.conf
- [x] Create `launch_scenario.yml`
- - [x] Wait for the scan.
- - [x] Check that the scan was executed at the right time.
- - [x] Create/Delete/modify the necessary files.
- - [x] Check alerts in `alerts.json`.
- - [x] Check alerts in `Elasticsearch`.

Regards,
Daniel Folch

**Comment by DFolchA** [comment]
Hello team,

I have created a python script that checks how much time has passed between scans and waits for the ongoing scans to finish.

I have also created new handlers to clean the osec.log file for convenience and created the `launch_environment.yml` playbook that:
- Prepares the environment (common task)
- Sets the scenario variables, eg frequency.
- Restarts Wazuh and cleans the ossec.log of the agents (handlers).
- Creates/modifies/deletes files (common tasks).
- Checks if the scans are producing at the right time (check_scheduled_scan.py).
- Waits for the ongoing scan to end (check_scheduled_scan.py).
- Checks if the alerts are correct (common tasks)

Working branch:
https://github.com/wazuh/wazuh-qa/tree/feature-553-scenario-210

Remaining tasks:
- [ ] Add logging to python script

Regards,
Daniel Folch
